### PR TITLE
FOGL-767 - Tests related fixes for Scheduler and Tasks. Necessitated due to change in dir structure

### PR DESCRIPTION
**Pull request by amarendra-dianomic**

### CODE REVIEW

File: tests/unit-tests/python/foglamp_test/services/core/test_scheduler_get_tasks.py [hunk 1]
```diff
@@ -58,9 +58,9 @@
 # TODO: To run this test,
 #       1) Do 'foglamp start' and note the management_port from syslog
 #       2) Change _m_port below with the management_port
-#       3) Execute this command: FOGLAMP_ENV=TEST pytest -s -vv tests/core/test_scheduler_get_tasks.py
+#       3) Execute this command: FOGLAMP_ENV=TEST pytest -s -vv tests/unit-tests/python/foglamp_test/services/core/test_scheduler_get_tasks.py
 _address = '0.0.0.0'
-_m_port = 41041
+_m_port = 41449
```
Comment (vaibhav-ScaleDB) [inline tests/unit-tests/python/foglamp_test/services/core/test_scheduler_get_tasks.py]: We will not hardcode management ports

File: tests/unit-tests/python/foglamp_test/services/core/test_scheduler.py [hunk 2]
```diff
@@ -81,6 +81,7 @@ class TestScheduler:
         await self.populate_test_data()  # Populate data in foglamp.scheduled_processes
 
         scheduler = Scheduler(_address, _m_port)
+        print(scheduler, _address, _m_port)
```
Comment (vaibhav-ScaleDB) [inline tests/unit-tests/python/foglamp_test/services/core/test_scheduler.py]: No prints

File: tests/unit-tests/python/foglamp_test/services/core/test_scheduler.py [hunk 3]
```diff
@@ -26,9 +26,9 @@
 # TODO: To run this test,
 #       1) Do 'foglamp start' and note the management_port from syslog
 #       2) Change _m_port below with the management_port
-#       3) Execute this command: FOGLAMP_ENV=TEST pytest -s -vv tests/core/test_scheduler.py
+#       3) Execute this command: FOGLAMP_ENV=TEST pytest -s -vv tests/unit-tests/python/foglamp_test/services/core/test_scheduler.py
```
Comment (vaibhav-ScaleDB) [inline tests/unit-tests/python/foglamp_test/services/core/test_scheduler.py]: Approach is yet to be decided to eliminate any manual steps

File: tests/unit-tests/python/foglamp_test/services/core/api/test_task.py [hunk 4]
```diff
@@ -62,15 +62,15 @@ class TestTask:
...
+        call(["scripts/foglamp", "start"])
         # TODO: Due to lengthy start up, now tests need a better way to start foglamp or poll some
         #       external process to check if foglamp has started.
         time.sleep(30)
 
     @classmethod
     def teardown_class(cls):
         from subprocess import call
-        call(["foglamp", "stop"])
+        call(["scripts/foglamp", "stop"])
```
Comment (vaibhav-ScaleDB) [inline tests/unit-tests/python/foglamp_test/services/core/api/test_task.py]: there is nothing called foglamp stop. We need to mock the foglamp start/stop commands to make it a pure unit test

File: tests/unit-tests/python/foglamp_test/services/core/api/test_scheduler.py [hunk 5]
```diff
@@ -52,15 +52,15 @@ class TestScheduler:
...
+        call(["scripts/foglamp", "start"])
         # TODO: Due to lengthy start up, now tests need a better way to start foglamp or poll some
         #       external process to check if foglamp has started.
         time.sleep(30)
 
     @classmethod
     def teardown_class(cls):
         from subprocess import call
-        call(["foglamp", "stop"])
+        call(["scripts/foglamp", "stop"])
```
Comment (vaibhav-ScaleDB) [inline tests/unit-tests/python/foglamp_test/services/core/api/test_scheduler.py]: there is nothing called foglamp stop. We need to mock the foglamp start/stop commands to make it a pure unit test

Comment (praveen-garg) [reply tests/unit-tests/python/foglamp_test/services/core/api/test_scheduler.py]: let's comment it, and add a TODO for now!

File: tests/unit-tests/python/foglamp_test/services/core/api/test_task.py (hunk 4 above)
Comment (praveen-garg) [reply tests/unit-tests/python/foglamp_test/services/core/api/test_task.py]: as above

File: tests/unit-tests/python/foglamp_test/services/core/test_scheduler.py (hunk 3 above)
Comment (praveen-garg) [reply tests/unit-tests/python/foglamp_test/services/core/test_scheduler.py]: Yes, add a TODO. but let's atleast have the ability to run these automated tests with manual intervention; 

i.e.find and set core mgt port, start/ stop foglamp etc.

**Comment by amarendra-dianomic** [comment]
@praveen-garg @vaibhav-ScaleDB changes done per feedback. Please check.

**Comment by praveen-garg** [comment]
@amarendra-dianomic  in setupclass we call(["scripts/foglamp", "start"]) 
will this not change the `core port`?

**Comment by amarendra-dianomic** [comment]
@praveen-garg these are two separate set of tests - integration tests (with api) and unit tests. We have to do foglamp start/stop with api tests only.

**Comment by praveen-garg** [comment]
Gotcha! Yupe!

**Comment by amarendra-dianomic** [comment]
@praveen-garg all fixed, pl check.

**Comment by amarendra-dianomic** [comment]
@praveen-garg changes done per feedback.

**Pull request merged**